server/repository/src/db_diesel/report_row.rs

```diff
@@ -205,4 +209,39 @@ mod test {
             );
         }
     }
+
+    #[actix_rt::test]
+    async fn report_can_upsert_all_fields() {
+        let (_, connection, _, _) = setup_all("report_can_upsert", MockDataInserts::none()).await;
```

Comment:
Was wondering why not add form_schemas through MockData ? 
There is an old issue to document mock data and how it's used in test, it's meant to make tests a bit easier to write

Reply:
Great test though, especially with the new data types